Source/WebCore/dom/Element.cpp

```diff
@@ -3590,8 +3590,6 @@ void Element::focus(const FocusOptions& options)
         return;
     }
 
-    document->updateContentRelevancyForScrollIfNeeded(*this);
```

Comment:
Can this be removed?

Reply:
It does not seem to make a difference (anymore?) when I remove it, looking at the wpt test results. However, if we want to play it safe, I can undo this part of the PR, let me know.

Reply:
I meant, can we remove the function definition for `updateContentRelevancyForScrollIfNeeded` altogether? not just the usage of it

Reply:
Do you mean like the first try by Nikos (this had several test fails)? :
https://github.com/WebKit/WebKit/pull/22064

Some code has to instruct the scrolled/focused to content that it is no longer skipped, otherwise the skipped content check in LocalFrameView::scrollRectToVisible will cause the operation to not go through, this is what updateContentRelevancyForScrollIfNeeded does. Let me know though if that is not what you meant.

Reply:
OK sounds good, can you update the FIXME on top of `void Document::updateContentRelevancyForScrollIfNeeded(const Element& scrollAnchor)` to either point to a new bug or remove it?

It might possible there's a better place to put this code now that scroll anchoring is implemented.

Reply:
Yeah, I was planning to remove the FIXME but forgot it before creating the new PR, will fix before landing.